EligibilityCheckOrchestrator/index.ts

```diff
@@ -160,6 +163,17 @@ export const handler = function*(
       );
     });
 
+    // If the provided fiscalCode is a testing one and the DSU is ELIGIBLE,
+    // the process is aborted.
+    fromNullable(process.env.TEST_FISCAL_CODES)
+      .map(_ => _.split(","))
+      .filter(_ => _.includes(fiscalCode))
```

Comment:
can we put this code into its own method ie. 

`const isTestFiscalCode = (fiscalCode: f, env = process.env) => ...`

Reply:
Fixed in 3c861e3